scripts/docker/buildpack-deps/Dockerfile.ubuntu.clang.ossfuzz

```diff
@@ -91,35 +92,38 @@ RUN set -ex; \
 	rm -rf /usr/src/libprotobuf-mutator
 
 # EVMONE
+WORKDIR cd /usr/src
 RUN set -ex; \
-	cd /usr/src; \
-	git clone --branch="v0.10.0" --recurse-submodules https://github.com/ethereum/evmone.git; \
-	cd evmone; \
-	mkdir build; \
-	cd build; \
-	CXX=clang++ cmake -G Ninja -DBUILD_SHARED_LIBS=OFF -DCMAKE_INSTALL_PREFIX="/usr" ..; \
+	git clone --branch="v0.11.0" --recurse-submodules https://github.com/ethereum/evmone.git
+WORKDIR evmone/build
+RUN	CXX=clang++ cmake -G Ninja -DBUILD_SHARED_LIBS=OFF -DCMAKE_INSTALL_PREFIX="/usr" ..; \
 	ninja; \
 	ninja install/strip; \
 	rm -rf /usr/src/evmone
+WORKDIR /solidity
 
 # gmp
+WORKDIR cd /usr/src
 RUN set -ex; \
-    # Replace system installed libgmp static library
-    # with sanitized version. Do not perform apt
-    # remove because it removes mlton as well that
-    # we need for building libabicoder
-    rm -f /usr/lib/x86_64-linux-gnu/libgmp.*; \
-    rm -f /usr/include/x86_64-linux-gnu/gmp.h; \
-    cd /usr/src; \
-    wget -q 'https://gmplib.org/download/gmp/gmp-6.2.1.tar.xz' -O gmp.tar.xz; \
-    test "$(sha256sum gmp.tar.xz)" = "fd4829912cddd12f84181c3451cc752be224643e87fac497b69edddadc49b4f2  gmp.tar.xz"; \
-    tar -xf gmp.tar.xz; \
-    cd gmp-6.2.1; \
-    ./configure --prefix=/usr --enable-static=yes; \
+	git clone --depth 1 --branch "gmp-6.2.1" https://github.com/gmp-mirror/gmp-6.2.git; \
+	tar -czf gmp-6.2.tar.gz gmp-6.2
+RUN test "$(sha256sum gmp-6.2.tar.gz)" = "b719f304ba51ddc13c76d8a51b739e5eeac5ac6062cb7e345fa2cf7d3d67bbea  gmp-6.2.tar.gz"; \
+	rm gmp-6.2.tar.gz
+	# Replace system installed libgmp static library
+	# with sanitized version. Do not perform apt
+	# remove because it removes mlton as well that
+	# we need for building libabicoder
+RUN	rm -f /usr/lib/x86_64-linux-gnu/libgmp.*; \
+	rm -f /usr/include/x86_64-linux-gnu/gmp.h
+WORKDIR gmp-6.2
+RUN autoreconf -i; \
+	./configure --prefix=/usr --enable-static=yes --enable-maintainer-mode; \
     make -j; \
-    make install; \
-    rm -rf /usr/src/gmp-6.2.1; \
-    rm -f /usr/src/gmp.tar.xz
+    make check; \
+    make install
+RUN rm -rf /usr/src/gmp-6.2.1; \
+    rm -f /usr/src/gmp-6-2.tar.gz
+WORKDIR /solidity
 
 # libabicoder
 RUN set -ex; \
```

Comment:
@bshastry I think this needs an update. Looks like [The upstream repo](https://github.com/mmalvarez/Yul-Isabelle) has ~60 new commits (only https://github.com/mmalvarez/Yul-Isabelle/commit/9adc71880aa28675f96f24733dd09d3b3a1318ed seems missing) over @ekpyron's fork, though it also stopped being updated 3 years ago. Do we actually still need a personal fork here?

Reply:
This is fine as is - I'd not call the repo you're referring to "upstream" - that'd be https://github.com/ethereum/Yul-Isabelle these days really - but we don't need to touch this at all - this was academic work, so there was never a clear repo separation between the formal abi encoder and proper yul semantics, but the abi encoder formalization that's relevant here hasn't been updated from the state used here.

Reply:
Hmmm... well, ok. The thing that bothers me the most here is the use of a personal repo, as I know from experience that these tend to disappear as people owning them leave the project. I don't think this is a big risk in your case, but I'd prefer to avoid those in general. I was actually going to suggest to maybe transfer the repo to the `ethereum` org, but looks like there actually is one. What do you think about pushing your changes there and switching to it just for the peace of mind? Would that be a lot of effort?

Reply:
It doesn't really belong in that repo in the first place, the abi encoder formalization is independent of the yul formalization.
But yeah, in this case here: this was a time-boxed Grants project between Mario at Consensus and myself - his personal repo is not really more upstream, we just reused the old repo before we moved the Yul formalization part to the ethereum-repo.

None of this is critical in any case - if this vanishes for whatever reason, we can just drop this here anyways, so as I said: I'd just not touch this until it breaks and if it does probably just drop it.